trustpoint/est/views.py

```diff
@@ -424,18 +424,31 @@ def deserialize_pki_message(self, data: bytes) -> tuple[CredentialRequest | None
         :raises ValueError: If deserialization fails.
         """
         try:
-            csr = x509.load_der_x509_csr(data, default_backend())
+            if b'-----BEGIN CERTIFICATE REQUEST-----' in data:
```

Comment:
I love this approach to dynamically accept all common CSR formats :)
I would propose changing this check to `if b'CERTIFICATE REQUEST-----' in data:` to support the alternative `BEGIN NEW CERTIFICATE REQUEST` form as well: https://stackoverflow.com/questions/28628744/is-there-a-spec-for-csr-begin-headers